frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -691,7 +691,7 @@ const exampleList: React.FC<MyProps> = () => {
 
 | Parameter Name | Description |
 | -------------- | ----------- |
-| `groupVersionKind` | the resource group/version/kind to represent |
+| `groupVersionKind` | group, version, kind of k8s resource {@link K8sGroupVersionKind} is preferred alternatively can pass reference for group, version, kind which is deprecated i.e `group~version~kind` {@link K8sResourceKindReference}. |
```

Comment:
We should document how to handle core resource like pod without a group as well

Reply:
Hi Sam, I'm not sure what handle core resource like pod without a group is referring to. Can you give me an example of how to document this?